packages/@aws-cdk/aws-appsync/lib/schema.ts

```diff
@@ -106,4 +160,25 @@ export class Schema {
     this.addToSchema(Lazy.stringValue({ produce: () => type.toString() }));
     return type;
   }
+
+  /**
+   * Set the root types of this schema if they are defined.
+   *
+   * For example:
+   * schema {
+   *   query: Query
+   *   mutation: Mutation
+   *   subscription: Subscription
+   * }
+   */
+  private declareSchema(): string {
+    if (!this.query && !this.mutation && !this.subscription) return '';
```

Comment:
put condition of if into braces 

rationale: consistency with code base & readability